test-requirements.txt

```diff
@@ -9,7 +9,8 @@ sure
 pure-sasl
 twisted[tls]; python_version >= '3.5'
 twisted[tls]==19.2.1; python_version < '3.5'
-gevent>=1.0
+gevent>=1.0; platform_machine != 'i686' and platform_machine != 'win32'
```

Comment:
When we used `Gevent` in the code?

Reply:
it's part of the `python-driver` is support running with gevent, and there are a few test for that